docs/transactions/iti41.md

```diff
@@ -1,35 +1,228 @@
+
+* Code of [medication](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medication.html) should match the code of medication of the [MTP referenced by the PRE](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-document-medicationprescription.html) (`CHEMEDEPRMedicationRequest.treatmentplan`).      
+* If [lot number](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medication.html) (`CHEMEDEPRMedication.batch`) is specified in referenced MTP, [the one in the PRE](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medicationrequest.html) (`CHEMEDEPRMedicationRequest.medication[x]:medicationReference.batch`) should match it.
+#### PADV
+##### PADV against a provisional PRE
+A "provisional PRE" is a prescription for which the [`CHEMEDEPRMedicationRequest.status`](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medicationrequest.html) is `SUBMITTED` or `PROVISIONAL` and for which a PADV `OK` has not been completed yet. 
+* Medication cannot be changed.
+* Dosage instructions cannot be changed.
+* Substitution cannot be disallowed if the existing dispense includes one.
+* Observation code `REFUSE` cannot be issued.
+#####  PADV `CANCEL`
+* Not allowed against DIS or other PADV documents.
+
+#### DIS
+* If [lot number](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medication.html) (`batch`) is specified in referenced [MTP(`treatmentPlan`) or PRE (`prescription`)](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medicationdispense.html), then it should match the one in the DIS.
+##### DIS against PRE or MTP:
+* Medication cannot be different if substitution is not allowed by PRE. _This rule might be extended in the future_. 
+* Dispense cannot occur after the end of validity of the PRE document. _This rule might be extended in the future_.
+* Dispense cannot occur after the end of the treatment if specified in the prescription item. _This rule might be extended in the future_.
+* If referenced PRE is not provisional, it should be [dispensable](#FIXME).
+* List of active [ingredients](https://build.fhir.org/ig/hl7ch/ch-emed//StructureDefinition-ch-emed-medication-medicationdispense-definitions.html#Medication.ingredient) should be the same than the one in the referenced [PRE](https://build.fhir.org/ig/CARA-ch/ch-emed-epr/StructureDefinition-ch-emed-epr-medicationdispense.html) (`prescription`). 
+
```

Comment:
The following check is also performed:

- If the treatment has been prescribed (at least one PRE document has been successfully submitted for it), then any DIS document targeting this treatment must reference one of its prescriptions.